lib/vehicle.rb

```diff
@@ -22,4 +25,14 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
-end
+
+  def plate_type
```

Comment:
This does work, but this plate_type should be set upon successful registration of the vehicle so we need to tie this method into the register_vehicle method.